### DISCUSSION TITLE
Angst

### DISCUSSION THREAD

**Discussion by Raytwo** [Raytwo/ARCropolis#364] [Announcements]
### New feature(s)
* Display the total size of loaded mods in the mod manager (thanks @blu-dev)

### Bugfixe(s)
* Fix crashes and infinite loadings appearing seemingly at random (thanks @blu-dev)
* Fix UI filter not sorting properly in the mod manager (thanks @blu-dev)
* Reimplement missing ``mp4`` extension support for webm (thanks @jozz024)
* Fix an issue where special taunts (Kid Icarus/Metal Gear Solid/Starfox) would play the wrong audio clips (thanks @Coolsonickirby)
* For plugin developers:
  * Fix a crash when using the ``arc:/`` virtual filesystem (thanks @jozz024)

### Special thanks
* The ARCropolis breaker gang
* Angst

# Read this
Skyline is not shipped with ARCropolis anymore.
Please read [this issue](https://github.com/Raytwo/ARCropolis/issues/157) to know how to proceed from now on.

<hr /><em>This discussion was created from the release <a href='https://github.com/Raytwo/ARCropolis/releases/tag/v3.5.0'>Angst</a>.</em>

**Comment by jozz024** [comment]
![image](https://user-images.githubusercontent.com/74272560/173134670-4333a26c-02a1-4090-8a98-a6db8ac8f94a.jpeg)
